### ISSUE TITLE
Init for a config directory in root broken

### ISSUE THREAD

**Issue by mshish** [gphotosuploader/gphotos-uploader-cli#172]
I opened a PR to update the docker image for gphotos-uploader to v1.0.3 (rfgamaral/docker-gphotos-uploader#13). It appears that at some point between v0.8.3 and the current release initializing a config in the root directory no longer works.

```
# gphotos-uploader-cli init --config=/config
[fatal]  unlinkat //config: device or resource busy
```

**Comment by pacoorozco** [comment]
Hi @mshish 

The problem is not the root folder, It's a permission problem, given that `gphotos-uploader-cli init --config=/config` will try to remove `/config` folder and recreate it again.

When you try to do that inside a docker, you don't have rights to remove `/config`... 

One possible workaround would be to remove folder contents instead of the folder itself. Let me test it.

**Comment by pacoorozco** [comment]
Fixed in the related PR, next version v1.0.4 should work as expected. Thanks for raising it up @mshish
